plugins/faustwp/readme.txt

```diff
@@ -1,18 +1,18 @@
-=== Faust ===
+=== Faust.js ===

```

Comment:
The trademark symbol, `™`, is intentionally not being used here as it might interfere with someone searching for the Faust WordPress plugin. This was approved by legal.

Note that this maps to the title on the [plugin title on .org](https://wordpress.org/plugins/faustwp/).